logger/message/audit/entry.go

```diff
@@ -31,6 +31,10 @@ type Entry struct {
 	DeploymentID string    `json:"deploymentid,omitempty"`
 	Time         time.Time `json:"time"`
 	Event        string    `json:"event"`
+
+	// Class of audit message - S3, admin ops, bucket management
+	Category string `json:"category,omitempty"`
```

Comment:
Should this be a typed category? We can perhaps call this Type? Like AuditType ?

Reply:
> Should this be a typed category? We can perhaps call this Type? Like AuditType ?

Done